apps/remix-ide/src/app/tabs/compile-tab.js

```diff
@@ -124,7 +124,9 @@ class CompileTab extends CompilerApiMixin(ViewPlugin) { // implements ICompilerA
       path: [],
       pattern: []
     })
-    this.currentFile = await this.call('fileManager', 'file')
+    try {
+      this.currentFile = await this.call('fileManager', 'file')
+    } catch (error) {}
```

Comment:
So If I am right, It will never show error for any failure on `this.call('fileManager', 'file')`

Reply:
well it does this on activation, a lot of the times there is no file selected when compiler is activated. it's not user action.

Reply:
should we assign an empty string in catch? @bunsenstraat

Reply:
no, it's already set by the init of the base class
https://github.com/ethereum/remix-project/blob/a11fe6b8d81cd68edff88b5ab32be6074077f721/apps/solidity-compiler/src/app/compiler-api.ts#L44